index.html

```diff
@@ -636,6 +636,526 @@ <h3>Criteria and Criterion</h3>
+            DID Method specification suitable for evaluation,
+            including any specified variant such as testnet, mainnet, 
+            etc.
+            </li>            
+          </ol>
+        </section><!-- Criteria.Examples.Method-->
+        <section>
+          <h4>Criteria.Examples.Referent</h4>
+          <ol>
+            <li>Evaluations MAY refer to open ended responses from other 
+            evaluations. If present, this referent MUST be listed as a
+            possible response in another criteria’s evaluation. In this 
+            manner, a first criteria can elicit structural elements to
+            be separately evaluated in subsequent criteria. This 
+            “structure-variable” pattern allows the Rubric to support 
+            criteria that matches the structural complexity of the 
+            Method in question.
+            </li>
+            <li>If present, the Referent MUST be in the second column
+            </li>
+            <li>The column title MUST be appropriate for the referred 
+            structure, e.g., “Layer” or “Governing body”, as understood 
+            by the criteria which establishes the structural elements.
+            </li>
+          </ol>
+        </section><!-- Criteria.Examples.Referent-->          
+        <section>
+          <h4>Criteria.Examples.Responses</h4>
+          <ol>
+            <li>Example evaluations MUST include at least one response, 
+            and may include multiple columns when appropriate. Each 
+            column allows for responses to particular system elements 
+            such as “Specification”, “Network”, and “Registry”.
+            </li>
+            <li>
+            Column names such as "Specification", "Network", and 
+            "Registry" may be abbreviated as "Spec.", "Net.", and "Reg."
+            </li>
+            <li>
+            The number and labels for response columns MUST be proposed 
+            by the initial PR and will be finalized when the criteria
+            moves from PROVISIONAL to accepted.
+            </li>
+            <li>Each column MUST contain an entry that communicates the 
+            evaluator’s judgment of the best response to the question for
+            this method for that element (Spec, Reg, Net).
+            </li>
+            <li>For improved comparability across different methods, 
+            responses SHOULD be based on the possible responses listed 
+            in the criteria. However, evaluators MAY provide any 
+            appropriate response, based on their own judgment
+            </li>
+            <li>
+            Responses MAY be a variant of the Possible Responses, to 
+            capture a nuance beyond those listed. For example, one might
+            append a “+” or a “-”, e.g., A+, or even provide two 
+            different responses when different situations merit a 
+            distinction, e.g., A/D. The reasoning behind these variants 
+            &nbps;
```

Comment:
think this may be causing an error
```suggestion
            &nbsp;
```

Reply:
not sure a non-breaking space is actually needed . . .

Reply:
Turns out it is, or the italized SHOULD which follows ends up looking like there is no space.

Reply:
interesting. regardless, the html character code is misspelled.

Reply:
and also interesting that none of the other italicized SHOULDs require it

Reply:
I notice that no other `SHOULD` starts a line of the HTML source (following some number of spaces).  I wonder if changing to the other source-style (i.e., with the tags indented and their content un-indented) would be a viable workaround? Yes, it's a bigger change than the unexpected `&nbsp;`... Editors' choice, I guess.

Reply:
It was a weird standout. I'll keep my eye open for fixing in a more consistent way, but for now, I'm going to take you up on the editor's choice so we can get the PR merged.

Reply:
I think it's fine to keep the &nbsp,  but even in that case, [this suggestion](https://github.com/w3c/did-rubric/pull/49/files#r706419263) should be merged.